### ISSUE TITLE
TreeTable doesn't render due to missing markup exception with Wicket 7.0

### ISSUE THREAD

**Issue by dashorst** [wicketstuff/wicket1.5-tree#1]
```
org.apache.wicket.markup.MarkupNotFoundException: Markup not found for Component: [TreeTableItem [Component id = 0]]
    at org.apache.wicket.Component.internalRender(Component.java:2347)
    at org.apache.wicket.Component.render(Component.java:2307)
    at org.apache.wicket.extensions.markup.html.tree.table.SideColumnsView.onRender(SideColumnsView.java:104)
    at org.apache.wicket.Component.internalRender(Component.java:2379)
    at org.apache.wicket.Component.render(Component.java:2307)
    at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1407)
    at org.apache.wicket.MarkupContainer.renderAll(MarkupContainer.java:1604)
    at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1579)
    at org.apache.wicket.MarkupContainer.renderAssociatedMarkup(MarkupContainer.java:691)
    at org.apache.wicket.markup.html.panel.AssociatedMarkupSourcingStrategy.renderAssociatedMarkup(AssociatedMarkupSourcingStrategy.java:77)
    at org.apache.wicket.markup.html.panel.PanelMarkupSourcingStrategy.onComponentTagBody(PanelMarkupSourcingStrategy.java:112)
    at org.apache.wicket.Component.internalRenderComponent(Component.java:2564)
    at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1543)
    at org.apache.wicket.Component.internalRender(Component.java:2379)
    at org.apache.wicket.Component.render(Component.java:2307)
    at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1407)
    at org.apache.wicket.MarkupContainer.renderAll(MarkupContainer.java:1604)
    at org.apache.wicket.Page.onRender(Page.java:879)
    at org.apache.wicket.markup.html.WebPage.onRender(WebPage.java:143)
    at org.apache.wicket.Component.internalRender(Component.java:2379)
    at org.apache.wicket.Component.render(Component.java:2307)
    at org.apache.wicket.Page.renderPage(Page.java:1018)
    at org.apache.wicket.request.handler.render.WebPageRenderer.renderPage(WebPageRenderer.java:124)
    at org.apache.wicket.request.handler.render.WebPageRenderer.respond(WebPageRenderer.java:236)
    at org.apache.wicket.util.tester.BaseWicketTester$LastPageRecordingPageRendererProvider$1.respond(BaseWicketTester.java:2741)
    at org.apache.wicket.core.request.handler.RenderPageRequestHandler.respond(RenderPageRequestHandler.java:175)
    at org.apache.wicket.request.cycle.RequestCycle$HandlerExecutor.respond(RequestCycle.java:865)
    at org.apache.wicket.request.RequestHandlerStack.execute(RequestHandlerStack.java:64)
    at org.apache.wicket.request.cycle.RequestCycle.execute(RequestCycle.java:265)
    at org.apache.wicket.request.cycle.RequestCycle.processRequest(RequestCycle.java:222)
    at org.apache.wicket.request.cycle.RequestCycle.processRequestAndDetach(RequestCycle.java:293)
    at org.apache.wicket.util.tester.BaseWicketTester.processRequest(BaseWicketTester.java:717)
    at org.apache.wicket.util.tester.BaseWicketTester.processRequest(BaseWicketTester.java:656)
    at org.apache.wicket.util.tester.BaseWicketTester.processRequest(BaseWicketTester.java:628)
    at org.apache.wicket.util.tester.BaseWicketTester.startPage(BaseWicketTester.java:1261)
    at org.apache.wicket.util.tester.BaseWicketTester.startPage(BaseWicketTester.java:1234)
    at org.apache.wicket.extensions.markup.html.tree.TreeTablePageTest.pageRenders(TreeTablePageTest.java:10)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:606)
    at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
    at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
    at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
    at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
    at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
    at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
    at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
    at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
    at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
    at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
    at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
    at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
    at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
    at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
    at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
    at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
    at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
    at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
    at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
```

**Comment by dashorst** [comment]
Fixed in 7.0.1

**Comment by smartuadk** [comment]
Hi Dashorst, 

I am also facing this same issue in Wicket 8.


		treeImport = new TreeTable("statustreeImport", new DefaultTreeModel(rootNode), columnsImport);
		initImportStatus();
		treeImport.setRootLess(true);
		treeImport.setOutputMarkupId(true);
		addOrReplace(treeImport);



=========================================================================


ERROR | Exception occured:
org.apache.wicket.markup.MarkupNotFoundException: Markup not found for Component: [TreeTableItem [Component id = 0]]
	at org.apache.wicket.Component.internalRender(Component.java:2267)
	at org.apache.wicket.Component.render(Component.java:2227)
	at org.apache.wicket.extensions.markup.html.tree.table.MiddleColumnsView.onRender(MiddleColumnsView.java:189)
	at org.apache.wicket.Component.internalRender(Component.java:2296)
	at org.apache.wicket.Component.render(Component.java:2227)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1454)
	at org.apache.wicket.MarkupContainer.renderAll(MarkupContainer.java:1654)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1629)
	at org.apache.wicket.MarkupContainer.renderAssociatedMarkup(MarkupContainer.java:798)
	at org.apache.wicket.markup.html.panel.AssociatedMarkupSourcingStrategy.renderAssociatedMarkup(AssociatedMarkupSourcingStrategy.java:77)
	at org.apache.wicket.markup.html.panel.PanelMarkupSourcingStrategy.onComponentTagBody(PanelMarkupSourcingStrategy.java:112)
	at org.apache.wicket.Component.internalRenderComponent(Component.java:2491)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1593)
	at org.apache.wicket.Component.internalRender(Component.java:2296)
	at org.apache.wicket.Component.render(Component.java:2227)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1454)
	at org.apache.wicket.MarkupContainer.renderAll(MarkupContainer.java:1654)
	at org.apache.wicket.Page.onRender(Page.java:858)
	at org.apache.wicket.markup.html.WebPage.onRender(WebPage.java:126)
	at org.apache.wicket.Component.internalRender(Component.java:2296)
	at org.apache.wicket.Component.render(Component.java:2227)
	at org.apache.wicket.Page.renderPage(Page.java:998)
	at org.apache.wicket.request.handler.render.WebPageRenderer.renderPage(WebPageRenderer.java:124)
	at org.apache.wicket.request.handler.render.WebPageRenderer.respond(WebPageRenderer.java:236)
	at org.apache.wicket.core.request.handler.RenderPageRequestHandler.respond(RenderPageRequestHandler.java:202)
	at org.apache.wicket.request.cycle.RequestCycle$HandlerExecutor.respond(RequestCycle.java:914)
	at org.apache.wicket.request.RequestHandlerExecutor.execute(RequestHandlerExecutor.java:65)
	at org.apache.wicket.request.cycle.RequestCycle.execute(RequestCycle.java:282)
	at org.apache.wicket.request.cycle.RequestCycle.processRequest(RequestCycle.java:253)
	at org.apache.wicket.request.cycle.RequestCycle.processRequestAndDetach(RequestCycle.java:221)
	at org.apache.wicket.protocol.http.WicketFilter.processRequestCycle(WicketFilter.java:275)
	at org.apache.wicket.protocol.http.WicketFilter.processRequest(WicketFilter.java:206)
	at org.apache.wicket.protocol.http.WicketFilter.doFilter(WicketFilter.java:299)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:320)
	at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.invoke(FilterSecurityInterceptor.java:126)
	at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.doFilter(FilterSecurityInterceptor.java:90)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:118)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:137)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:111)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.authentication.rememberme.RememberMeAuthenticationFilter.doFilter(RememberMeAuthenticationFilter.java:158)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:158)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:63)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.authentication.www.BasicAuthenticationFilter.doFilterInternal(BasicAuthenticationFilter.java:204)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.header.HeaderWriterFilter.doHeadersAfter(HeaderWriterFilter.java:92)
	at org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:77)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:56)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:105)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334)
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:215)
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:178)
	at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:358)
	at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:271)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:543)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:688)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:615)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:818)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1623)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)


-------------------------------------------------------------------

Can you please let me know what are the fixes for this issue?
